docs/faq.md

```diff
@@ -473,3 +473,44 @@ in a Swift string `"grey_typeText("foob\bar")"` to type "fooar". How do I use ba
 
 For Swift, the backspace escape character is `\u{8}`. You need to add that in your string to be typed for
 Swift. For example, To type "fooar", you should use `grey_typeText("foob\u{8}ar")`
+
+**Does EarlGrey support finding react-native elements?**
+
+Yes. By default [all touchable elements](https://facebook.github.io/react-native/docs/accessibility.html)
+are accessible. A button with the `accessibilityLabel` prop set can be found by `grey_accessibilityLabel`.
+For other elements, `accessible: true` must be set otherwise `grey_accessibilityLabel` will not match on them
+even if they have a label. Finding by label will not match on non-accessible elements. This matches the behavior
```

Comment:
Hey bootstraponline, please take a look at an alternate wording (this is required because EarlGrey does match elements which VoiceOver may not access because they were disabled from a different point in hierarchy):

Finding by label only works on accessible elements to reflect the fact that accessibility attributes (barring accessibility identifier which is actually used for automation) make sense only in case of accessibility elements, i.e isAccessibilityElement must be YES.